model.py

```diff
@@ -4648,14 +4648,13 @@ def calculate_opds_entries(self, verbose=True):
             VerboseAnnotator,
         )
         _db = Session.object_session(self)
-        simple = AcquisitionFeed.single_entry(_db, self, Annotator,
-                                              force_create=True)
-        if simple is not None:
-            self.simple_opds_entry = unicode(etree.tostring(simple))
```

Comment:
Work.simple_opds_entry is actually set within single_entry, to an intermediate value. The value actually returned by single_entry is a feed that has been further annotated with license information (in this case, the provider of the LicensePool). That's not what we want in Work.simple_opds_entry -- we want the intermediate value with no provider info.